css-ruby/Overview.bs

```diff
@@ -160,14 +160,10 @@ Ruby-specific 'display' Values</h3>
 	authors must map document language elements to ruby elements;
 	this is done with the 'display' property.
 
-	<table class="propdef">
-		<tr>
-			<th>Name:
-			<td>display
-		<tr>
-			<th><a href="#values">New Values</a>:
-			<td>ruby | ruby-base | ruby-text | ruby-base-container | ruby-text-container
-	</table>
+	<pre class="propdef">
```

Comment:
Put a "partial" class on this as well, so it doesn't try to claim to be defining the 'display' property.